README.md

```diff
@@ -3,12 +3,18 @@
 ## Procedure
 
 1. Create a repository with new agent anywhere outside the [42ITy
-   organization](https://github.com/42Ity)
+   organization](https://github.com/42Ity).
```

Comment:
stick to some one casing of `42[Ii][Tt]y` in the paragraph plz :)
and also check if github URLs are okay with the specified (or any) casing - e.g. I found that it matters for branch names